docs/readthedocs/source/doc/Nano/Overview/tensorflow_inference.md

```diff
@@ -80,7 +80,7 @@ from torchmetrics.classification import MulticlassAccuracy
 
 q_model = model.quantize(precision='int8',
                          accelerator=None,
-                         calib_dataset= train_dataset,
+                         x= train_dataset,
```

Comment:
we are having a fix on https://github.com/intel-analytics/BigDL/pull/6925 for this.

Reply:
Ok, then that's fine!